### PR TITLE
fix raptor env vals

### PR DESCRIPTION
**Pull request by AymenFJA**
This will fix issue #2602

### CODE REVIEW

File: src/radical/pilot/raptor/worker_mpi.py [hunk 1]
```diff
@@ -519,8 +519,8 @@ def _dispatch(self, task):
                'RP_GTOD'            : os.environ['RP_GTOD'],
                'RP_PROF'            : os.environ['RP_PROF'],
                'RP_PROF_TGT'        : os.environ['RP_PROF_TGT'],
-               'RP_RANKS'           : 1,  # dispatch_mpi will oveerwrite this
-               'RP_RANK'            : 0,  # dispatch_mpi will oveerwrite this
+               'RP_RANKS'           : str(1),  # dispatch_mpi will oveerwrite this
+               'RP_RANK'            : str(0),  # dispatch_mpi will oveerwrite this
                })
```
Comment (andre-merzky) [inline src/radical/pilot/raptor/worker_mpi.py:524]: Another option would be to allow other types as env values and to explicitly convert them when setting the env variables.  That would provide some additional safety against user level errors.  What do you think?

Comment (AymenFJA) [reply src/radical/pilot/raptor/worker_mpi.py:524]: @andre-merzky : so something like this: 

```python
        for k, v in task['description'].get('environment', {}).items():
            os.environ[k] = str(v)
```

Comment (andre-merzky) [reply src/radical/pilot/raptor/worker_mpi.py:524]: Yes, indeed.

**Comment by AymenFJA** [comment]
@andre-merzky Done, comment addressed in the last commit.

**Review by andre-merzky — APPROVED** [review]
Thanks @AymenFJA , that LGTM!

**Pull request merged**